lxd/devices.go

```diff
@@ -180,10 +186,74 @@ func deviceNetlinkListener() (chan []string, chan []string, chan device.USBEvent
 				chUSB <- usb
 			}
 
+			if (props["SUBSYSTEM"] == "char" || props["SUBSYSTEM"] == "block") && udevEvent {
```

Comment:
@Lilyjjo related to my comment above, why is the `udevEvent` boolean needed rather than just checking the `SUBSYSTEM` property alone? Thanks

Reply:
@tomponline 
this is related to us changing the netlink socket filter to read both kernel uevents as well as udev events, we added the string.prefix("libudev") to determine whether an event string was a kernel uevent or a udev event, and then our code is only able to properly handle identifying the product/vendor ID from udev events, hence the if statement.

Reply:
@splett2 ok thanks, it would be great to have a comment to that effect where you set the udevEvent var to true so that this reasoning is clear in the future. Thanks